testsuites/syncgateway/functional/tests/test_db_online_offline_resync.py

```diff
@@ -42,6 +45,11 @@ def test_bucket_online_offline_resync_sanity(params_from_base_test_setup, sg_con
 
     start = time.time()
 
+    if x509_cert_auth:
```

Comment:
rather than having if condition, you can directly add persist_cluster_config_environment_prop(cluster_conf, 'x509_certs', x509_cert_auth) as you are assigning x509_cert_auth to true/false in the parameters

Reply:
done